docs/content/deployment/guides/kubernetes/how-to-migrate-your-instance.mdx

```diff
@@ -38,7 +56,17 @@ kubectl scale deploy $WEBSERVER_DEPLOYMENT_NAME --replicas=0
 kubectl scale deploy $DAEMON_DEPLOYMENT_NAME --replicas=0
 ```
 
-5. Run a Kubernetes Job with the `dagster instance migrate` command. You can check if the migration succeeded by inspecting the the resulting Pod of the Job.
+---
+
+## Step 3: Migrate the Dagster instance
+
+Next, run a Kubernetes Job with the `dagster instance migrate` command:
+
+```shell
+dagster instance migrate
+```
+
+Check if the migration succeeded by inspecting the resulting Pod of the Job:
```

Comment:
I don't think this part is quite right - the "helm template" command below is what causes the "dagster instance migrate" command to be run, but the new formatting and copy implies that you first run 'dagster instance migrate' yourself somehow and then run the "helm template" to check that it worked.

We could give an actual example of how you check the pod worked? Something like

"kubectl get pods -l job-name=dagster-instance-migrate" to get the pod name
then
"kubectl describe pod <pod name>" once you have the pod name

Or we could re-format things to make it clear that the helm template command is what does the migration

Reply:
Took a shot at this - let me know what you think